gnomad_qc/v3/resources/release.py

```diff
@@ -58,9 +58,9 @@ def release_ht_path(
     :rtype: str
     """
     if public:
-        return f"gs://gnomad-public/release/{release_version}/ht/{data_type}/gnomad.{data_type}.r{release_version}.sites.ht"
+        return f"gs://gnomad-public/release/{release_version}/ht/{data_type}/gnomad.{data_type}.v{release_version}.sites.ht"
```

Comment:
```suggestion
        return f"gs://gnomad-public-requester-pays/release/{release_version}/ht/{data_type}/gnomad.{data_type}.v{release_version}.sites.ht"
```
(though this won't work for v3.0)

Reply:
All resources are currently not working as expected. There is a good chance things will be moving around more so I am waiting for a full and separate PR to address all of these issues.